routes.py

```diff
@@ -8,5 +10,10 @@ def hello_world():
     return "<h1>Hello, World!</h1>"
 
 
+@app.route("/foo")
+def rand_element():
+    return f"<h1>{randint(1, 20)}</h1>"
```

Comment:
A couple of things with this line.

1: The 1 and the 20 are called magic numbers. Picture reading a file with a thousand lines and then a random unexplained integer in the middle. What is it? Is it safe to change it? Who knows. They should be defined as constants with descriptive names in UPPER_SNAKE_CASE at the top of this file or in a dedicated constants.py file and imported here.

2: We should probably update this to return a rendered template like current master is doing. You have options for getting you branch up to date with master. You can rebase you branch of latest master locally and force push to your remote branch or you can merge master into your branch.

Reply:
How do we un-outdate this comment? Will adding this comment do it?

Reply:
I see. The associated line no longer exists. Let me know what you want to talk about how to do 2.

Reply:
Definitely going to need help with number 2. I'm totally lost in "parallel universes" of git right now.